### ISSUE TITLE
Entry field "options" is being overridden by a function "options" !!!!

### ISSUE THREAD

**Issue by manekinekko** [marmelab/ng-admin#529]
Hi,

I have an entity which has a field name "options". It appears that this field is being overridden by a function white the same name. 
For instance, take the code below, the "options" field gets transformed into a function whose prototype is Empty.

```
{
   "thisField": "isOK",
    "options": [{
            "label": "OK",
            "value": "1",
            "correct": true,
            "_id": {
                "$oid": "55895fe4cf6e556a532801b8"
            }
        }]
}
```

Any idea?
Thx.

![image](https://cloud.githubusercontent.com/assets/1699357/8354684/2fb096ce-1b4a-11e5-8feb-6f6eb9fad63e.png)

**Comment by karantan** [comment]
Your "options" data in your JSON object is not the same "options" that you point in the picture.

ng-admin by default (or Restangular which is responsible for the req & res) do not transform any fields.

take a look at this [link](https://github.com/mgonto/restangular#addresponseinterceptor) for more details about Restangular.

**Comment by fzaninotto** [comment]
Duplicate of #273, will be fixed one day by #278

**Comment by manekinekko** [comment]
I renamed the "options" field and now it's working.
